components/elm/src/biogeochem/AllocationMod.F90

```diff
@@ -2510,7 +2518,7 @@ subroutine Allocation3_PlantCNPAlloc (bounds            , &
                 cpool_to_xsmrpool(p) = 0.0_r8
 
                 ! storage pool turnover
-                xsmrpool_turnover(p) = max(xsmrpool(p) - mr*xsmr_ratio*dt , 0.0_r8) / (10.0_r8*365.0_r8*secspday)
+                xsmrpool_turnover(p) = max(xsmrpool(p) - mr*xsmr_ratio*dt , 0.0_r8) / (nsc_rtime(ivt(p))*365.0_r8*secspday)
```

Comment:
@qzhu-lbl @rgknox It appears this fix is needed in master. Correct?

Reply:
yes